library/core/src/array/mod.rs

```diff
@@ -80,7 +76,7 @@ where
 /// assert_eq!(array, None);
 /// ```
 #[inline]
-#[unstable(feature = "array_from_fn", issue = "89379")]
+#[stable(feature = "array_from_fn", since = "1.61.0")]
 pub fn try_from_fn<F, R, const N: usize>(cb: F) -> ChangeOutputType<R, [R::Output; N]>
```

Comment:
Note for libs-api: This would be the first stable usage of the https://doc.rust-lang.org/nightly/std/ops/trait.Residual.html mechanism (the other things like `Iterator::try_find` are still unstable) so it probably deserves extra careful consideration.